qutebrowser/utils/usertypes.py

```diff
@@ -266,6 +266,7 @@ class Question(QObject):
                  For user_pwd, a default username as string.
         title: The question title to show.
         text: The prompt text to display to the user.
+        yank_text: The prompt text available to prompt-yank command.
```

Comment:
If it's always an URL, I'd prefer this to be called `url`. I'm just not sure if there's something else other than URLs to yank in prompts which would make sense, but the rest of your code talks about URLs as well.